packages/xo-server-sdn-controller/src/index.js

```diff
@@ -704,7 +704,7 @@ class SDNController extends EventEmitter {
           pool: object.$pool.name_label,
         })
 
-        if (find(this._newHosts, { $ref: object.$ref }) === undefined) {
+        if (this._newHosts.find(_ => _.$ref === object.$ref) === undefined) {
```

Comment:
You can use [`Array#some`](https://devdocs.io/javascript/global_objects/array/some) for this case.